build.gradle

```diff
@@ -121,7 +121,7 @@ jacocoTestReport {
     }
 }
 
-String version = '6.2.0'
+String version = '6.2.1'
```

Comment:
Can you not bump the version? I can take care of it in my upcoming PR. Thanks!

Reply:
When are you planning to bump this version? This is fixing the distribution build so we might want to bump it soon.

Reply:
By EOD. We are having too many tags for 1 fixes. Trying to club 3 changes together.

Reply:
I changed it back. Please prioritize the version bump so that we could update our build Jenkinsfiles to start testing. Thanks.